kafka-connect/src/main/scala/net/manub/embeddedkafka/connect/EmbeddedKafkaConnect.scala

```diff
@@ -0,0 +1,106 @@
+package net.manub.embeddedkafka.connect
+
+import java.io.File
+
+import org.apache.kafka.common.utils.Time
+import org.apache.kafka.connect.connector.policy.ConnectorClientConfigOverridePolicy
+import org.apache.kafka.connect.runtime.isolation.Plugins
+import org.apache.kafka.connect.runtime.rest.RestServer
+import org.apache.kafka.connect.runtime.standalone.{
+  StandaloneConfig,
+  StandaloneHerder
+}
+import org.apache.kafka.connect.runtime.{Connect, Worker, WorkerConfig}
+import org.apache.kafka.connect.storage.MemoryOffsetBackingStore
+import org.apache.kafka.connect.util.ConnectUtils
+
+import net.manub.embeddedkafka.{
+  EmbeddedKafka,
+  EmbeddedKafkaConfig,
+  EmbeddedKafkaSupport
+}
+
+import scala.jdk.CollectionConverters._
+
+/**
+  * Helper trait for running Kafka Connect server.
+  * Use `.startConnect` to start the server.
+  */
+trait EmbeddedKafkaConnect
+    extends EmbeddedKafkaConnectSupport[EmbeddedKafkaConfig]
+    with EmbeddedKafka {
+  override protected[embeddedkafka] val connectConfig =
+    new EmbeddedConnectConfigImpl
+}
+
+private[embeddedkafka] trait EmbeddedKafkaConnectSupport[
+    C <: EmbeddedKafkaConfig
+] {
+  this: EmbeddedKafkaSupport[C] =>
+
+  protected[embeddedkafka] def connectConfig: EmbeddedConnectConfig[C]
+
+  /**
+    * Start a Kafka Connect server and pass a block of code that can
+    * operate while the server is active.
+    *
+    * @param connectPort the Kafka Connect port for the REST API to listen on.
+    * @param offsets     the file to store offset data in.
+    * @param extraConfig additional Kafka Connect configuration (overwrite existing keys in
+    *                    default config)
+    * @param block       the code block that will executed while the Kafka Connect server
+    *                    is active. Once the block has been executed the server will be stopped.
+    */
+  def startConnect[T](
+      connectPort: Int,
+      offsets: File,
```

Comment:
I'd use `java.nio.file.Path` instead, as it's already used by the library sometimes; also it works nicely with `Files.createTempFile` ;)

Reply:
Done!